### PR TITLE
[DSK] Implement Grab the Prize

### PR DESCRIPTION
**Pull request by Jamaninja**
Part of #12534

### CODE REVIEW

File: Mage.Sets/src/mage/cards/g/GrabThePrize.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    public GrabThePrize(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.SORCERY}, "{1}{R}");
+
+        // As an additional cost to cast this spell, discard a card.
+        this.getSpellAbility().addCost(new DiscardCardCost());
+
+        // Draw two cards.
+        this.getSpellAbility().addEffect(new DrawCardSourceControllerEffect(2));
+
+        this.getSpellAbility().addEffect(new ConditionalOneShotEffect(
```
Comment (JayDi85) [inline Mage.Sets/src/mage/cards/g/GrabThePrize.java:33]: Miss comment with full ability text

**Review by JayDi85 — APPROVED** [review]
All fine, only one comment miss

File: Mage.Sets/src/mage/cards/g/GrabThePrize.java (hunk 1 above)
Comment (Jamaninja) [reply Mage.Sets/src/mage/cards/g/GrabThePrize.java:33]: Sorry, this was my first card. Should I include the comment in a commit the next time I upload a card?

Comment (JayDi85) [reply Mage.Sets/src/mage/cards/g/GrabThePrize.java:33]: You can edit and add commit to your version, it will be available in that PR too.

**Review by FdjNL — APPROVED** [review]

**Pull request merged**